tao_compiler/mlir/xla/ral/context/pdll_util.cc

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "tensorflow/compiler/mlir/xla/ral/context/pdll_util.h"
+
+#include "tensorflow/compiler/mlir/xla/ral/ral_logging.h"
+
+namespace tao {
+namespace ral {
+
```

Comment:
better to have emit* functions here, too.

Reply:
It'll introduce llvm dependency, which it's not what we want in bridge side.